### ISSUE TITLE
Menu bar disappears randomly

### ISSUE THREAD

**Issue by Neischa** [tullamods/Dominos#687]
Hi, since start of Dragonflight I noticed Menu bar often disappears and I have to reload UI to get it back. No error message so no idea what the issue is, sorry:(
Retail, latest addon version...

**Comment by Tuller** [comment]
Have you tried disabling your addons other than Dominos?

**Comment by Neischa** [comment]
Not yet, I am trying to find some relation between this and some other action I am doing as I don't want to blindly play without addons for whole day. I will let you know when i find something. I thought maybe someone else also have this issue or u will know what it could be just like that:) Sorry!

**Comment by Neischa** [comment]
Update - it didnt happen for me in last 5 days at all, but happened today like 4 times in 2 hours, no idea why:(

**Comment by Neischa** [comment]
Update - still hapenning couple of times per day, now I found that when this happens and menu bar buttons disappear it affects only enabled buttons - I have enabled all but "Character Info" button. When I enable this Character Info button while others are "bugged" and not visible, that Character Info button becomes visible so its not like menu bar completely disappears, but rather only currently visible buttons disappear (until I /reload). However disabling/enabling missing buttons have no effect so on the end I have to /reload anyway, but at least I found something which maybe helps you?

**Comment by Neischa** [comment]
Update - I joined Brawl today and Menu buttons disappeared, had to reload UI and when I got out, it happened again, two Brawls in a row. Thats maybe something, right?

**Comment by Neischa** [comment]
Another update - when I die and jump down from the ship, it always hide the menu buttons, i just tried like 10 times, it always happen.

**Comment by Neischa** [comment]
Update - using SELFIE camera always makes micro menu buttons disappear, I just tested it 5 times and always I enter selfie mode buttons are gone! Same happens during world quest where you take pictures of animals etc...

**Comment by Tuller** [comment]
10.0.15 should fix this

**Comment by Neischa** [comment]
I tested it with SELFIE camera and it works, micro menu disappears when I enter SELFIE mode, but then reappears. Strange is that my other action bars are still visible even in SELFIE mode, but micro menu is not. I tried to check "Show with override UI", but it didn't do anything. Is it supposed to work like this?

Btw. thanks a lot for fixing it, I will update it once more when new world q appears and I will have a chance to test it more thoroughly:)
